sirepo/pkcli/job_cmd.py

```diff
@@ -217,6 +218,8 @@ def _change_message_too_big(msg):
     if type(msg) != bytes:
         msg = pkjson.dump_bytes(msg)
     if len(msg + b'\n') > job.cfg.max_message_bytes:
+        # TODO (gurhar1133): Do we need to change things so that we don't have to send COMPLETED for
+        # this error case?
         msg = pkjson.dump_bytes(PKDict(
                 state=job.COMPLETED,
                 error='Data was too large',
```

Comment:
This message should be more explanatory: Response is too large to send.  

The data is too large, but what type of data is it? It's a response. It could be too large to be stored. This indicates a problem in the communication channel, not a problem with the simulation producing too much data.